tests/src/CLRTest.Execute.Bash.targets

```diff
@@ -120,14 +120,39 @@ then
   exit 0
 fi
       ]]></BashCLRTestEnvironmentCompatibilityCheck>
-      <BashCLRTestGCSimulatorSkipCondition Condition="$(GCSimulatorRun) != 'true' And '$(IsGCSimulatorTest)' == 'true'"><![CDATA[
-echo "Skipping execution because GCSimulator tests are not enabled"
-exit 0
-      ]]></BashCLRTestGCSimulatorSkipCondition>
-      <BashCLRTestGCLongTestSkipCondition Condition="$(GCLongRunning) != 'true' And '$(IsLongRunningGCTest)' == 'true'"><![CDATA[
-echo "Skipping execution because long-running GC tests are not enabled"
-exit 0
+
+      <BashCLRTestGCLongTestSkipCondition Condition="'$(IsLongRunningGCTest)' != 'true'"><![CDATA[
```

Comment:
It would also be nice to move this logic to another file like what was done for R2R in https://github.com/dotnet/coreclr/blob/master/tests/src/CLRTest.CrossGen.targets to say CLRTest.GC.targets

And append all the runtime logic to BashCLRTestPreCommands and CLRTestBatchPreCommands. You don't need a target to do this.  You can just use a property-group